composeApp/src/main/kotlin/io/github/couchtracker/tmdb/TmdbModule.kt

```diff
@@ -0,0 +1,17 @@
+package io.github.couchtracker.tmdb
+
+import app.moviebase.tmdb.Tmdb3
+import io.github.couchtracker.utils.lazyEagerModule
+
+val TmdbModule = lazyEagerModule {
+    single {
+        Tmdb3 {
+            tmdbApiKey = TmdbConfig.API_KEY
+            useTimeout = true
+            maxRequestRetries = 3
+        }
+    }
+    single {
```

Comment:
If you want you can use [constructor DSL](https://insert-koin.io/docs/reference/koin-core/dsl-update/).

While it makes no real difference now, it helps on those instances where you want to inject dependencies.

i.e.
```kotlin
single(::TmdbMemoryCache)
```

Just FYI